src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -626,6 +626,11 @@ public Builder addInitializerBlock(CodeBlock block) {
       return this;
     }
 
+    public Builder addRawInitializerBlock(CodeBlock block) {
```

Comment:
happy to change the name if maintainers prefer (e.g. `addRawCodeBlock` as suggested by @bondolo)